api/schemas/views.py

```diff
@@ -125,9 +125,6 @@ class RegistrationSchemaFormBlocks(JSONAPIBaseView, generics.ListAPIView):
     def get_queryset(self):
         schema_id = self.kwargs.get('schema_id')
         schema = get_object_or_error(RegistrationSchema, schema_id, self.request)
-        RegistrationSchema.objects.get_latest_versions()
-        if schema.schema_version != RegistrationSchema.objects.get_latest_version(name=schema.name, only_active=False).schema_version:
-            raise exceptions.ValidationError('Registration schema must be active.')
```

Comment:
I'd think we'd need to be able to return even inactive schema blocks, if we were going to render old registrations?